### ISSUE TITLE
How can one display images side by side in a post

### ISSUE THREAD

**Issue by i-mat** [cotes2020/jekyll-theme-chirpy#1123]
I'm trying to show a comparison between two photos in my post which is why I want to display them side-by-side. 
![image](https://github.com/cotes2020/jekyll-theme-chirpy/assets/37477590/67f742db-4d1e-444a-834e-b56d1ed55243)
Here is how the two images are placed currently. Help would be much appreciated, thanks!

**Comment by VanniPrashar** [comment]
Can I fix this please

**Comment by i-mat** [comment]
You can fix it? 

> Can I fix this please

**Comment by i-mat** [comment]
> Can I fix this please

You can fix it? THX, what is your suggestion?

**Comment by sukhjinder-kumar** [comment]
You can use HTML wrappers like the following, however, this will create a scroll on mobile devices.
```HTML
<p>
  <img src="..." width="49%" />
  <img src="..." width="49%" />
</p>
```
You can also try putting images in columns of a table.

Reference - https://bobbyhadz.com/blog/github-display-images-side-by-side-in-readme#:~:text=The%20easiest%20way%20to%20display,each%20column%20stores%20an%20image.

**Comment by cotes2020** [comment]
Just add the `normal` class to each image.

```markdown
![img-a](/url-img-a){: .normal .me-2  }
![img-b](/url-to-img-b){: .normal }
...
```

Note: The `me-2` above is the margin class of Bootstrap, which is used to add a gap between the images, you can choose the right size. See also: <https://getbootstrap.com/docs/5.2/utilities/spacing/#margin-and-padding>

**Comment by chgio** [comment]
I can't get the closing fix to work on v7.1.1... Any future-proof ideas?

**Comment by kungfux** [comment]
Here is one of the possible solutions depending on the exact behaviour you are trying to achieve. Consider creating a new [discussion](https://github.com/cotes2020/jekyll-theme-chirpy/discussions/new?category=q-a) if you still have questions.

```md
{: .d-flex .justify-content-center }
![img-a](https://chirpy-img.netlify.app/commons/avatar.jpg){: .normal .mw-50 .me-2}
![img-b](https://chirpy-img.netlify.app/commons/avatar.jpg){: .normal .mw-50 }
```
